packages/backend/src/server/WellKnownServerService.ts

```diff
@@ -39,6 +39,7 @@ export class WellKnownServerService {
 
 		const allPath = '/.well-known/*';
 		const webFingerPath = '/.well-known/webfinger';
+		const wfphPath = '/.well-known/protocol-handler';
```

Comment:
I think you don't need to store as variable (about webfinger, it used by multiple locations, but wfphPath only used by fastify.get)